lib/RexLint.php

```diff
@@ -23,7 +24,12 @@ final class RexLint
     public static function runFromWeb()
     {
         $lintErrors = self::lintPaths();
-        $jsonErrors = self::validateAddOnsPackageYml();
+
+        $jsonErrors = [];
+        // package.json schema was bogus in earlier redaxo core versions
+        if (rex_version::compare(rex::getVersion(), '5.15.1-dev', '>=')) {
```

Comment:
5.15.1 ist ja schon released. Bei 5.15.1 kommt aus dem Check hier also `true` raus. Daher:

```suggestion
        if (rex_version::compare(rex::getVersion(), '5.15.2-dev', '>=')) {
```

Und wir sollten im Core dann die Version anheben.